src/main/java/xbot/common/simulation/IWebotsSensor.java

```diff
@@ -0,0 +1,7 @@
+package xbot.common.simulation;
+
+import org.json.JSONObject;
+
+public interface IWebotsSensor {
```

Comment:
Thoughts on this being more generically about simulator than webots? Eg `SimulatableSensor`

Reply:
Good idea - there's nothing in here Webots-specific, so I'll change to a more generic name.